SPARQLWrapper/Wrapper.py

```diff
@@ -177,1169 +183,1184 @@
 #######################################################################################################
 
 
-class SPARQLWrapper(object):
+class QueryResult(object):
```

Comment:
Is there a reason for moving this up in the file? It makes the diff quite noisy, to do a proper review I have to copy it back to where it was.

Reply:
230a029

undo and quote returned type hints including QueryResult in SPARQLWrapper